experimental/traffic-portal/src/app/shared/generic-table/generic-table.component.html

```diff
@@ -21,7 +21,7 @@
 			<fa-icon [icon]="caretIcon" class="caret" [ngClass]="{'rotate': showMenu}"></fa-icon>
 		</button>
 		<mat-menu #menu="matMenu">
-			<button mat-menu-item *ngFor="let c of columns; orderBy:'colDef.headerName'" (click)="toggleVisibility($event, c.getColId())">
+			<button mat-menu-item *ngFor="let c of columns" (click)="toggleVisibility($event, c.getColId())">
```

Comment:
Since it was invalid, the console was throwing warning (which is how I found it). Changing it to `c.headerName` makes it throw an error instead.
```
NG0303: Can't bind to 'ngForOrderBy' since it isn't a known property of 'button' (used in the 'GenericTableComponent' component template).
1. If 'button' is an Angular component and it has the 'ngForOrderBy' input, then verify that it is a part of an @NgModule where this component is declared.
2. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component.
```

Reply:
Yeah I'm seeing that too, looks like they dropped support for ordering in NgFor in favor of just having you pass the collection in the order you want it. I'm not sure why, since that's not trivial for asynchronous data sources.